json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -55,6 +55,7 @@ class WPCOM_JSON_API_GET_Site_Endpoint extends WPCOM_JSON_API_Endpoint {
 		'site_migration'    => '(array) Data about any migration into the site.',
 		'is_fse_active'     => '(bool) If the site has Full Site Editing active or not.',
 		'is_fse_eligible'   => '(bool) If the site is capable of Full Site Editing or not',
+		'is_core_site_editor_enabled'	=> '(bool) If the site has the core site editor enabled.',
```

Comment:
Noting that we'll need to address those tabs in a follow-up PR as they generate phpcs errors. I'll take care of that.

Reply:
Thanks a bunch, appreciate that!

Just curious, do we just need to line up all the `=> $value`'s after our ridiculously longer name set them off balance?  Or is it something else regarding this?

Reply:
> do we just need to line up all the => $value's after our ridiculously longer name set them off balance? Or is it something else regarding this?

That's it, yes, combined with avoiding the use of tabs for aligning. Otherwise `phpcs` complains.